buildozer/buildozer_test.sh

```diff
@@ -447,6 +447,42 @@ function test_remove_package_attribute() {
   [ $(wc -c < "$root/pkg/BUILD") -eq 0 ] || fail "Expected empty file"
 }
 
+function test_remove_if_equal_label() {
+  in='go_library(
+    name = "edit",
+    shared_library = ":local",  # Suffix comment.
+)'
+  run "$in" 'remove_if_equal shared_library :local' '//pkg:edit'
+  assert_equals 'go_library(name = "edit")'
+}
+
+function test_remove_if_equal_ident() {
+  in='go_library(
+    name = "edit",
+    flag = True,
+)'
+  run "$in" 'remove_if_equal flag True' '//pkg:edit'
+  assert_equals 'go_library(name = "edit")'
+}
+
+function test_remove_if_equal_string() {
+  in='go_library(
+    name = "edit",
+    flag = "True",
+)'
+  run "$in" 'remove_if_equal flag True' '//pkg:edit'
+  assert_equals 'go_library(name = "edit")'
+}
+
+function test_remove_if_equal_string_attr_string() {
+  in='go_library(
+    name = "edit",
+    toolchain = "something",
+)'
+  run "$in" 'remove_if_equal toolchain something' '//pkg:edit'
+  assert_equals 'go_library(name = "edit")'
+}
+
```

Comment:
You should add a negative test as well (where the value is not equal)

Reply:
done